api/src/main/java/io/strimzi/api/kafka/model/KafkaResources.java

```diff
@@ -203,4 +203,15 @@ public static String zookeeperServiceName(String clusterName) {
     public static String zookeeperHeadlessServiceName(String clusterName) {
         return clusterName + "-zookeeper-nodes";
     }
+
+    /**
+     * Returns the name of the Cruise Control {@code Deployment} for a {@code Kafka} cluster of the given name.
+     * This {@code Deployment} will only exist if {@code Kafka.spec.cruiseControl} is configured in the
+     * {@code Kafka} resource with the given name.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @return The name of the corresponding Cruise Control {@code Deployment}.
+            */
+    public static String cruiseControlDeploymentName(String clusterName) {
+        return clusterName + "-cruise-control";
+    }
```

Comment:
The deployment name can be grabbed from the CruiseControlResources [1]

[1] https://github.com/kyguy/strimzi-kafka-operator/blob/cruise-control-deployment/api/src/main/java/io/strimzi/api/kafka/model/CruiseControlResources.java#L13-L20

Reply:
Oh right, missed that class. I will switch to that.